Playbooks/playbook-PanoramaCommitConfiguration.yml

```diff
@@ -3,7 +3,7 @@ version: -1
 name: PanoramaCommitConfiguration
 fromversion: 4.0.0
 description: "Commit PAN-OS Panorama or Firewall Configuration.\nIf specified
-  as Panorama, will also push the Policies to the specified Device Group in the instance "
+  as Panorama, will also push the Policies to the specified Device Group in the instance. playbook is deprecated please use pan-os-commit-configuration instead"
```

Comment:
please add the deprecated tag

Reply:
it failed to commit with the deprecated tag